apps/utopia-api.ru

```diff
@@ -0,0 +1,18 @@
+
+require 'utopia/controller'
+
+APP = Utopia::Controller.new(nil)
+
+class Controller < Utopia::Controller::Base
+	prepend Utopia::Controller::Actions
+	
+	on '*' do
```

Comment:
The endpoint must be `/` and respond to a `GET` request, so any other endpoint or request type must return 404.

See the Rack example for reference.

Reply:
Okay done.